### ISSUE TITLE
Adjust bai B5 11 02

### ISSUE THREAD

**Issue by Rolli1962** [csowada/openhab2-addons#7]
The Telegram of B5 11 02 is landing always in ebus-unresolved.csv.
Result of check: Telegram is always a Byte longer as configured in vaillant-bai00-configuration.json and (as least in my system) always 64.
Example of an telegram:
2020-02-04 21:13:58;"10";"08";"B5 11";"01 02 8A 00 06 03 3C 96 46 8C 64 E9 00";

Adjusted configuration tested in a custom-configuration is working (added static 64):

		{
            "label":    "Operational Data of Burner Control Unit to Room Control Unit",
            "id":       "boiler.control.getopdata.02",
            "command":  "B5 11",

            "get": {
                "master": [
                    {"type": "static", "default": "02"}
                ],
                "slave": [
                    {"type": "static", "default": "03 3C 96 46"},
					{"name": "srv_water_target", "type": "data1c", "label": "Service water target temperature", "min": 0, "max": 100, "format":"%.1f°C"},
                    {"type": "static", "default": "64"}
                ]
            }
        }

**Comment by csowada** [comment]
Oh, I forgot to submit my message :-)

-------------------------------------------

The command is very rudimentary implemented. Could you try this version below? But Vaillant has two versions of the answer part, one with 6 bytes and one with 5 bytes. If it works I can include it in the core configurations. It is not tested!

```
"label":    "Operational Data of Burner Control Unit to Room Control Unit",
            "id":       "boiler.control.getopdata.02",
            "command":  "B5 11",

            "get": {
                "master": [
                    {"type": "static", "default": "02"}
                ],
                "slave": [
                    {"name":"hwcmode", "type": "uchar", "label":"HWC mode", "format":"%d",
                        "mapping": {"0":"disabled", "1":"on", "2":"off", "3":"auto"}},

                    {"name": "temp0_max", "type": "uchar", "label": "Sensor 0 target temperature (max)", "min": 0, "max": 100, "format":"%d°C"},
                    {"name": "temp0_target", "type": "data1c", "label": "Sensor 0 target temperature", "min": 0, "max": 100, "format":"%.1f°C"},

                    {"name": "srv_water_target_max", "type": "uchar", "label": "Service water target temperature (max)", "min": 0, "max": 100, "format":"%d°C"},
                    {"name": "srv_water_target", "type": "data1c", "label": "Service water target temperature", "min": 0, "max": 100, "format":"%.1f°C"},

                    {"name": "_unknown1", "type": "uchar", "label": "Unknown value"},
                ]
            }
        },
```

**Comment by Rolli1962** [comment]
Yes, is working well.
Had only to elemenate the last comma (of course last statement) of:

                    {"name": "_unknown1", "type": "uchar", "label": "Unknown value"},

**Comment by csowada** [comment]
Included in binding version 2.5.1-6
